CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.1] - 2021-05-17
```

Comment:
Update this date before I merge the PR

Reply:
Done here https://github.com/ArrowSphere/public-api-client/commit/f4e1f96cdd4cc2343d2e971def96b7a677419a49